packages/react-composites/src/composites/ChatComposite/adapter/ChatAdapter.ts

```diff
@@ -87,6 +103,18 @@ export interface ChatAdapterThreadManagement {
    *
    */
   loadPreviousChatMessages(messagesToLoad: number): Promise<boolean>;
+  /* @conditional-compile-remove-from(stable): FILE_SHARING */
+  /**
+   * Sets the uploaded files in ui state.
+   * @beta
+   */
+  uploadFiles?: (uploadedFiles: UploadedFile[]) => void;
+  /* @conditional-compile-remove-from(stable): FILE_SHARING */
+  /**
+   * Marks the uploaded files in ui state as complete.
+   * @beta
+   */
+  uploadsComplete?: () => void;
```

Comment:
I would recommend against this method. Each file transfer completes on its own, the UI should decide when to stop showing the file transfer UI / reenable chat messages based on whether all file transfers are complete.

Adding this method creates two ways of saying when all transfer are done (one via each individual `UploadedFile`, and one here). This can easily cause confusion (what happens if I call this before all files are transferred? What happens if I call this then call `uploadFiles` again?)